controller-manager/src/main/resources/templates/ServiceMonitor-console.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: enmasse-console
+  namespace: {{ index .Params "Namespace" }}
+  labels:
+    monitoring-key: middleware
+    app: enmasse
+spec:
+  selector:
+    matchLabels:
+      app: enmasse
+      component: console
```

Comment:
This needs to be `consoleservice` as the service created for the console is labelled `component: consoleservice`

Reply:
That's done.